src/ARToolkit/ARSceneView.cs

```diff
@@ -253,8 +253,10 @@ public Mapping.Camera OriginCamera
             get => _controller?.OriginCamera ?? throw new InvalidOperationException();  // Only null in design mode
             set
             {
-                if (_controller != null) 
+                if (_controller != null && !value.IsEqual(OriginCamera))
```

Comment:
Can't you just use `(value != OriginCamera)` ?
We're not as concerned about equality as we are same reference.